kconfig/config.go

```diff
@@ -24,7 +24,13 @@ func NewKeyValueMapFromSlice(values ...interface{}) KeyValueMap {
 	mapping := KeyValueMap{}
 
 	for _, value := range values {
-		str := fmt.Sprintf("%s", value)
+		var str string
+		switch t := value.(type) {
+		case string:
+			str = t
+		case int, uint, int64, uint64:
```

Comment:
It might be good to cover all the remaining integer types - (u)int(8|16|32).

Reply:
Good shout! Thanks, I've updated the PR. :)